src/main.ts

```diff
@@ -10,9 +10,11 @@ import recipeRouter from "./routes/recipe";
 const app = express();
 const config = loadConfig();
 
+app.use(express.json());
 app.use(logger);
 app.use(cors());
 
+
 app.use("/recipe", recipeRouter);
```

Comment:
You may want to add `app.use("/resources", uploadRoute)` (unless you planned something else)